crates/corro-agent/src/api/public/pubsub.rs

```diff
@@ -842,8 +844,8 @@ async fn forward_bytes_to_body_sender(
                         return;
                     }
                 } else {
-                    if let Err(e) = poll_fn(|cx| tx.poll_ready(cx)).await {
-                        warn!(%sub_id, error = %e, "body sender was closed or errored, stopping event broadcast sends");
+                    if tx.is_closed() {
+                        warn!(%sub_id, "body sender was closed, stopping event broadcast sends");
```

Comment:
The Sender in http_body_util no longer has poll_ready or any other way to determine if the receiving body is closed.